usaspending_api/api_contracts/contracts/v2/awards/federal_account_count.md

```diff
@@ -0,0 +1,24 @@
+FORMAT: 1A
```

Comment:
Since the URLs have changed for these endpoints, the file paths need to be changed as well to match. You'll need to make a new `count/` dir to store them